backend/glp/admin.py

```
@@ -0,0 +1,271 @@
+    autocomplete_fields = ('user',)
+
+    def has_add_permission(self, request):
+        return request.user.is_superuser
+
+    def has_change_permission(self, request, obj=None):
+        return False
+
+    def has_delete_permission(self, request, obj=None):
+        return False
+
+    def save_model(self, request, obj, form, change):
```

Ruff (0.16.1)

[warning] Unused method argument: `form`

(ARG002)